scripts/ci/cmake-v2/ci-ubuntu20.04-gcc10-mpich.cmake

```diff
@@ -20,13 +23,18 @@ ADIOS2_USE_SZ:BOOL=ON
 ADIOS2_USE_ZeroMQ:STRING=ON
 ADIOS2_USE_ZFP:BOOL=ON
 
+Python_ROOT_DIR:PATH=${PY_ROOT}
+Python_FIND_STRATEGY:STRING=LOCATION
+Python_FIND_FRAMEWORK:STRING=FIRST
+
 CMAKE_C_FLAGS:STRING=-Wall
 CMAKE_CXX_FLAGS:STRING=-Wall
 CMAKE_Fortran_FLAGS:STRING=-Wall
 
 MPIEXEC_MAX_NUMPROCS:STRING=${N2CPUS}
 ")
 
-set(CTEST_CMAKE_GENERATOR "Ninja")
+set(CTEST_TEST_ARGS PARALLEL_LEVEL 1)
+set(CTEST_CMAKE_GENERATOR "Unix Makefiles")
```

Comment:
Why using make

Reply:
@scottwittenburg

Reply:
Do you want `Ninja` for all these builds?  In `master` I see:

```
$ grep -irn "unix makefiles" scripts/ci
scripts/ci/cmake-v2/ci-el8-oneapi-serial.cmake:25:set(CTEST_CMAKE_GENERATOR "Unix Makefiles")
scripts/ci/cmake-v2/ci-ascent-nvhpc.cmake:28:set(CTEST_CMAKE_GENERATOR "Unix Makefiles")
scripts/ci/cmake-v2/ci-el8-gcc8-mpi.cmake:32:set(CTEST_CMAKE_GENERATOR "Unix Makefiles")
scripts/ci/cmake-v2/ci-el8-nvhpc222-mpi.cmake:33:set(CTEST_CMAKE_GENERATOR "Unix Makefiles")
scripts/ci/cmake-v2/ci-el8-oneapi-mpi.cmake:32:set(CTEST_CMAKE_GENERATOR "Unix Makefiles")
scripts/ci/cmake-v2/ci-el8-gcc8-serial.cmake:25:set(CTEST_CMAKE_GENERATOR "Unix Makefiles")
scripts/ci/cmake-v2/ci-el8-icc-mpi.cmake:32:set(CTEST_CMAKE_GENERATOR "Unix Makefiles")
scripts/ci/cmake-v2/ci-common.cmake:15:  if(CTEST_CMAKE_GENERATOR STREQUAL "Unix Makefiles")
scripts/ci/cmake-v2/ci-el8-icc-serial.cmake:25:set(CTEST_CMAKE_GENERATOR "Unix Makefiles")
scripts/ci/cmake-v2/ci-el8-nvhpc222-serial.cmake:26:set(CTEST_CMAKE_GENERATOR "Unix Makefiles")
```

As with the ctest `PARALLEL_LEVEL`, I'm not sure I see the pattern as to when make is used vs ninja.  Possibly just to test both?

Reply:
> Do you want Ninja for all these builds? In master I see:

The ones that were using Makefile you can leave it to keep using Makefile; new builds try to use Ninja.

Same thing for the parallel_level 1.

Note that mpich build do not use Makefile or parallel_level 1 in master.